extensions/pkg/controller/controlplane/genericactuator/actuator.go

```diff
@@ -67,8 +67,8 @@ type ValuesProvider interface {
 // the values provided by the given values provider.
 func NewActuator(
 	providerName string,
-	secretConfigs func(namespace string) []extensionssecretsmanager.SecretConfigWithOptions, shootAccessSecrets func(namespace string) []*gardenerutils.ShootAccessSecret,
-	exposureSecretConfigs func(namespace string) []extensionssecretsmanager.SecretConfigWithOptions, exposureShootAccessSecrets func(namespace string) []*gardenerutils.ShootAccessSecret,
+	secretConfigs func(namespace string) []extensionssecretsmanager.SecretConfigWithOptions, shootAccessSecrets func(namespace string) []*gardenerutils.TargetAccessSecret,
```

Comment:
As said before, I'd vote to drop this commit. For existing `Shoot`s, the `Secret`s have already `shoot-access` in their names. There is too much effort in making all this generic, and especially too much in making it consistent.